src/ToricVarieties/LineBundles/auxilliary.jl

```diff
@@ -0,0 +1,36 @@
+
+function command_string(v::AbstractNormalToricVariety, c::Vector{fmpz})
+    # Initialize a list of strings, which we will eventually join to form the command string
+    string_list = Vector{String}()
+    
+    # Define helper function
+    joincomma(list) = join([string(x) for x in list], ",")
+    
+    # create dictionary from "our" variables to safe variable for use in cohomCalg
+    coordinate_names = [string(x) for x in Hecke.gens(cox_ring(v))]
+    coordinate_dictionary = Dict{String, String}(coordinate_names[i] => "x"*string(i) for i = 1:length(coordinate_names))
```

Comment:
Much safer and simpler, thanks :-)

A minor suggestion:
```suggestion
    coordinate_dictionary = Dict{String, String}(coordinate_names[i] => "x$i" for i = 1:length(coordinate_names))
```
or even this (Julia figured out the correct types automatically)
```suggestion
    coordinate_dictionary = Dict(coordinate_names[i] => "x$i" for i = 1:length(coordinate_names))
```

Reply:
Thx. Done.